### PR TITLE
More Checkstyle fixes

### PR DESCRIPTION
**Pull request by dallinson**

### CODE REVIEW

**Comment by Mahir-Sparkess** [comment]
SpeedUp.Java Line 39: after player.speedup(false) there should be used = false. I also see it missing from main? However it's in my local code. I use it to remove the powerup when used.

File: core/src/main/java/com/mygdx/auber/Powerups/SpeedUp.java [hunk 1]
```diff
@@ -3,37 +3,43 @@
...
         if (taken && !used) {
             position.x = player.getX();
             position.y = player.getY();
             timer += Gdx.graphics.getDeltaTime();
 
-            if (timer < 10) {
+            if (timer < Config.SPEEDUP_TIME) {
                 player.speedUp(true);
             } else {
                 player.speedUp(false);
```
Comment (Mahir-Sparkess) [inline core/src/main/java/com/mygdx/auber/Powerups/SpeedUp.java:45]: used = true

File: core/src/main/java/com/mygdx/auber/Powerups/SpeedUp.java [hunk 2]
```diff
@@ -43,9 +49,10 @@ public void update(Player player) {
     }
```
Comment (dallinson) [inline core/src/main/java/com/mygdx/auber/Powerups/SpeedUp.java:49]: see the lines above here

Comment (dallinson) [reply core/src/main/java/com/mygdx/auber/Powerups/SpeedUp.java:49]: you'll have to expand the block

**Pull request merged**